docs/contributing/releasing-islandora.md

```diff
@@ -102,14 +103,11 @@ completed all the above steps and are absolutely certain the release is ready fo
 
 | Name         | Organization           | Address               | Code Signing Key Fingerprint | Key Id |
 |--------------|------------------------|-----------------------|---|:-:|
-| Danny Lamb   | Islandora Foundation   | dlamb at islandora.ca | 2D609DB0380A7637A6B72B328D7E7725D47A05FA | D47A05FA |
+| Danny Lamb   | Born-Digital   | hello at born-digital.com  | 2D609DB0380A7637A6B72B328D7E7725D47A05FA | D47A05FA |
 | Jared Whiklo | University of Manitoba | jwhiklo at gmail.com  | 9F45FC2BE09F4D70DA0C7A5CA51C36E8D4F78790 | D4F78790 |
 | Nick Ruest   | York University        | ruestn at yorku.ca    | 159493E15691C84D615B7D1B417FAF1A0E1080CD | 0E1080CD |
 | Seth Shaw   | University of Nevada, Las Vegas        | seth.shaw at unlv.edu    | 2FF65B22AFA7B2A57F054F89D160AA658DAE385F | D160AA658DAE385F |
-
-## Releasing PHP Code
```

Comment:
Missing this header!

Reply:
I think @seth-shaw-unlv  wanted it removed?

Reply:
I don't recall making that request.

Reply:
It was a larger re-org piece about releasing things separately vs all repos at once. It stands. If we push this through in an acceptable state, i can look at tweaking it to better communicate what is implied by individual releases.